joblib/memory.py

```diff
@@ -541,6 +543,21 @@ def _cached_call(self, args, kwargs, shelving=False):
 
         return (out, args_id, metadata)
 
+    @property
+    def func_code_info(self):
+        # 3-tuple property containing: func_code, source_file, first_line
```

Comment:
we should make it clear that `self.func_code_info` does not return a `code` object, but a  string containing the source code (as well as some other metadata related to it)